templates/kubernetes/terraform/modules/kubernetes/db_user/mysql/main.tf

```diff
@@ -0,0 +1,97 @@
+# Create/Update mysql db user with kubernetes job
+resource "random_string" "job_id" {
+  length  = 8
+  lower   = true
+  upper   = false 
+  special = false
+
+  keepers = {
+    password = var.db_app_password
+  }
+}
+
+resource "kubernetes_namespace" "db_ops" {
+  metadata {
+    name = "db-ops"
+  }
+}
+
+resource "kubernetes_secret" "db_create_users" {
+  metadata {
+    name      = "db-create-users"
+    namespace = "db-ops"
+  }
+
+  data = {
+    "create-user.sql" = <<-EOF
+      DROP USER IF EXISTS "${var.db_app_user}";
+      CREATE USER "${var.db_app_user}" IDENTIFIED BY "${var.db_app_password}";
+      GRANT ALL PRIVILEGES ON ${var.db_name}.* TO "${var.db_app_user}";
+EOF
+    "RDS_MASTER_PASSWORD" = var.db_master_password
```

Comment:
This will leave the master password as a k8s secret in this namespace indefinitely, which is not great. Any thoughts on how to avoid this?